projects/radare2/default.options

```diff
@@ -0,0 +1,4 @@
+[libfuzzer]
+close_fd_mask=3
```

Comment:
This means that the logs will be less informative (e.g. we won't see the text for assertion failures).
I highly recommend to get rid of excessive logging and not use this option. 
Not a blocker thought.

Reply:
Good point. I'll try to mute the unnecessary output directly from radare2. I did that in order to avoid being too verbose and slow down the fuzzing.